moon-store-mac-os/moon-store-mac-os/Views/Product/AddProductView+Edit.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  AddProductView+Edit.swift
+//  moon-store-mac-os
+//
+//  Created by Jose Luna on 1/2/25.
+//
+
+import SwiftUI
+import PhotosUI
+
+private enum Constants {
+    static let cornerRadiusSize: CGFloat = 4
+    static let createButtonWidth: CGFloat = 200
+    static let crossIcon: String = "xmark"
+    static let modalMinHeight: CGFloat = 450
+    static let modalMinWidth: CGFloat = 650
+}
+
+struct AddProductView: View {
+    @Environment(\.dismiss) private var dismiss
+    @ObservedObject private var viewModel = AddProductViewModel()
```

Comment:
Mantene el mismo formato 

```suggestion
    @ObservedObject private var viewModel: AddProductViewModel = .init()
```

Reply:
Done